tools/build_env/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# See README.md file for how this image is used.
+
+FROM selenium/node-chrome
```

Comment:
This is a large image - and I guess it doesn't include Firefox. Could this explain why some of the tests still fail?

Anyway, we can think about moving to a difference baseimage at some point in the future.